src/main/scala/Streamer.scala

```diff
@@ -15,22 +18,33 @@ class Streamer {
    */
   def start(ssc: StreamingContext, keyspace: String, table: String) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1561119350/issues/source?bid=2980033&fileBranchId=3019286#l19)